Data/sync/Sync.swift

```diff
@@ -449,7 +449,7 @@ extension Sync {
                     
                 // TODO: Needs favicon
                 if clientRecord == nil {
-                    recordType.coredataModelType?.add(rootObject: fetchedRoot, save: false, sendToSync: false, context: context)
+                    _ = recordType.coredataModelType?.add(rootObject: fetchedRoot, save: false, sendToSync: false, context: context)
```

Comment:
We should add `@discardableResult` to the `add` function here, so we don't need to explicitly ignore it (e.g. `_ = `

Reply:
Done. Interestingly, the warning does not go away. Appears to be a bug with Xcode.